protocol/streaming/constants.go

```diff
@@ -0,0 +1,12 @@
+package streaming
+
+const (
```

Comment:
nit: can we add some comments indicating this is for full node streaming? streaming is a little generic

Reply:
`protocol/streaming` is the code location we use for FNS specific codes; will add more comments in this file